backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java

```diff
@@ -1595,7 +1595,7 @@ private Condition whereColumnInSubquery(Column c, SelectSelectStep subQuery) {
         return row(pkey)
```

Comment:
Ah, I think we should here have the 'root' table instead of the refTable.
The root table has the primary key.
Having this inheritance join indeed fixes it but is more expensive because of all the joining. 
Something like getRefTable().getRootTable() would be best

Reply:
Yes you're right, thanks